src/main_menu.cpp

```diff
@@ -119,8 +119,8 @@ std::vector<int> main_menu::print_menu_items( const catacurses::window &w_in,
         }
         ret.push_back( utf8_width_notags( text.c_str() ) );
 
-        std::string temp = shortcut_text( iSel == i ? hilite( c_yellow ) : c_yellow, vItems[i] );
-        text += string_format( "[%s]", colorize( temp, iSel == i ? hilite( c_white ) : c_white ) );
```

Comment:
The following call at line 131 should be updated to use the default separator (remove `']'`) because the items are now not separated by `]` anymore.